tests/components/application-connector/test/compass-runtime-agent/testkit/third_party/machinebox/graphql/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need to use some custom version of this `machinebox/graphql` or we can just use it as a normal dependency?

Reply:
There was a reason for that but I don't remember it :(

Reply:
Ok